spec/game_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../lib/game.rb'
+
+RSpec.describe Game do
+  it 'returns "Draw" if both answers are the same' do
+    new_game = Game.new("Rock")
+    expect(new_game).to receive(:opponent_answer).and_return("Rock")
+    expect(new_game.result).to eq "Draw"
+  end
+
+  it 'returns "Lose" if the player has lost' do
+    new_game = Game.new("Rock")
+    expect(new_game).to receive(:opponent_answer).and_return("Paper")
```

Comment:
For all the tests in this file, I wonder if you wanted to use allow instead of expect. 
```rb
allow(new_game).to receive(:opponent_answer).and_return("Paper")
```
Allow sets it up, but expect verifies that it happens. Given that you already have an expectation in these tests, maybe you juts wanted to stub with that line, not set up an additional expectation.